ProjectCoimbra.UWP/Project.Coimbra/Pages/BandSongPage.xaml.cs

```diff
@@ -33,6 +33,7 @@ public BandSongPage()
 
             this.SetCustomEventHandlers();
 
+            //TODO: Display a loading bar when initializing this BandSongPage
```

Comment:
you add this as a new work item in backlog and add the heading of it in comment here